scripts/summary_stats.R

```diff
@@ -0,0 +1,204 @@
+# Run with Rscript
+
+library(dplyr)
+library(ggplot2)
+library(ggpubr)
+theme_set(theme_pubclean())
+
+table<-read.table("predictions_ratio.txt", header=TRUE)
+pdf("genomad_vir2_pred.pdf", width = 5, height = 5, pointsize = 8)
```

Comment:
the pdf name should be a parameter

Reply:
This is not part of the pipeline but extra scripts we used for benchmarking geNomad and VIRify. We want to put those script public somewhere